src/common/sorted-table/index.js

```diff
@@ -387,11 +389,11 @@ export default class SortedTable extends Component {
         <table className='table'>
           <thead className='thead-default'>
             <tr>
-              {groupedActions != null && <th>
+              {!isEmpty(groupedActions) && <th>
```

Comment:
You could use an intermediary variable: `const hasGroupedActions = !isEmpty(groupedActions)`.